_config.yml

```diff
@@ -1,8 +1,8 @@
 title: P-society
 email: psocietyiiitbh@gmail.com
-description: We are a bunch of undergrads who like to dabble into all things in computer science from
-             web dev to machine learning and from distributed systems to blockchain,
-             helping others along the way in kick-starting their journey of becoming a developer.
+description: We are a bunch of undergraduate students who like to dabble into all things computer science from
```

Comment:
> undergraduate students

that's quite a mouthful. what's wrong with undergrads?

Reply:
Okay will leave it at undergrads. Nothing wrong!